Automattic/wc-calypso-bridge
# Update Appearance task has_products logic

### Changes proposed in this Pull Request:



Currently, `import sample products` step is shown in Appearance task when the store has products that are created by the Headstart tool because we use `HeadstartProducts::has_products()` to detect if there is a product.

This PR changes to use `Products::has_products()` instead so`import sample products` step is not shown in Appearance task when there are Headstart products.

related: https://github.com/Automattic/wc-calypso-bridge/pull/1054#issuecomment-1480815534







### How to test the changes in this Pull Request:




1. Go to `WooCommerce > Home` and click on `Personalize your store` task from Task list
2. Confirm `import sample products` step is not shown.



### Other information:




### FOR PR REVIEWER ONLY:
